addons/sale/models/sale_order_line.py

```diff
@@ -60,6 +60,10 @@ class SaleOrderLine(models.Model):
         related='order_id.state',
         string="Order Status",
         copy=False, store=True, precompute=True)
+    tax_country_id = fields.Many2one(
+        comodel_name='res.country',
+        related='order_id.tax_country_id',
```

Comment:
```suggestion
        related='order_id.tax_country_id',
```
No need for the `comodel_name`, it's automatically deducted from the related field by the orm ;).